external/runtime/runtime.depproj

```diff
@@ -2,6 +2,9 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <!-- support cross-targeting by choosing a RID to restore when running on a different machine that what we're build for -->
+    <NugetRuntimeIdentifier Condition="'$(OSGroup)' == 'Windows_NT' AND '$(OSEnvironment)' != 'Windows_NT'">win7-x64</NugetRuntimeIdentifier>
```

Comment:
OSEnvironment as recently been removed. https://github.com/dotnet/corefx/pull/15728, Use RunningOnUnix property for these 2 conditions.